test/jdk/javax/swing/SwingUtilities/TestBadBreak/TestBadBreak.java

```diff
@@ -83,11 +84,14 @@ protected void paintComponent(Graphics g) {
                 label.setOpaque(true);
                 frame.getContentPane().add(label);
                 frame.setBounds(200, 200, 200, 90);
+                frame.setUndecorated(true);
+                frame.setLocationRelativeTo(null);
             }
         });
         robot.waitForIdle();
         SwingUtilities.invokeAndWait(() -> frame.setVisible(true));
         robot.waitForIdle();
+        robot.delay(1000);
```

Comment:
Why do you need this delay and then one more in between the "test"?

Reply:
This is between visible and destroy..The other one before start of the 2nd test which will crate 2nd frame..

Reply:
And why do you need both? The test does not use the robot, so the test can show all windows at once.

Reply:
BTW since the robot is not used it is unclear how the setUndecorated:true/false and setLocationRelativeTo() may affect the test.

Reply:
Since the text is rendered in JLabel and it's rendered into a BufferedImage, do we need the frame at all?
If no frame is shown, the test can be headless, can't it?

Reply:
It depends on what L&F needs to be tested. The Aqua for ex can be unavailable in headless.

Reply:
Is it the point of the test? The test does not seem to depend on a particular L&F.